zookeeper-contrib/zookeeper-contrib-fatjar/pom.xml

```diff
@@ -62,6 +62,10 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+    </dependency>
```

Comment:
The fatjar is used as a library. You shouldn't let this runtime dependency propagate for the library use case, so this should be omitted here.

```suggestion
```